api/tests/unit/application/error-manager_test.js

```diff
@@ -642,5 +644,45 @@ describe('Unit | Application | ErrorManager', function () {
         });
       });
     });
+
+    context('Locale errors', function () {
+      context('When receiving LocaleFormatError', function () {
+        it('instantiates a BadRequest error', function () {
+          // given
+          const error = new LocaleFormatError('zzzz');
+          sinon.stub(HttpErrors, 'BadRequestError');
+          const params = { request: {}, h: hFake, error };
+
+          // when
+          handle(params.request, params.h, params.error);
+
+          // then
+          expect(HttpErrors.BadRequestError).to.have.been.calledWithExactly(
+            'Given locale is in invalid format: "zzzz".',
+            'INVALID_LOCALE_FORMAT',
+            { locale: 'zzzz' }
+          );
+        });
+      });
+
+      context('When receiving LocaleNotSupportedError', function () {
+        it('instantiates a BadRequest error', function () {
+          // given
+          const error = new LocaleNotSupportedError({ locale: 'nl-BE' });
```

Comment:
Suite à notre dernière mob ensemble on n'utilise plus d'autres propriétés que locale :

```suggestion
          const error = new LocaleNotSupportedError('nl-BE');
```